.circleci/build.sh

```diff
@@ -7,19 +7,33 @@ set -e
 echo "==> Validate composer"
 composer validate --ansi --strict
 
+[ -d build ] && echo "==> Remove existing build directory" && chmod -Rf 777 build && rm -rf build
+
 echo "==> Initialise Drupal site"
 composer create-project drupal-composer/drupal-project:8.x-dev build --no-interaction
 
 echo "==> Add additional dev dependencies"
 composer --working-dir=build require --dev dealerdirect/phpcodesniffer-composer-installer:^0.5
 
-echo "==> Copy module code"
+echo "==> Start inbuilt PHP server in $(pwd)/build/web"
```

Comment:
this code allows to run the same site build process locally, so it is possible to do development of this module by running `./.circleci/build.sh`

Reply:
Such a beautiful way